contributor_docs/access.md

```diff
@@ -1,3 +1,5 @@
+<!-- What our commitment to access means for contributors and users of the library. -->
```

Comment:
Hi @natdecker, please take a look at the one-line description of the access statement and share any feedback if you might have. Thanks!